### PR TITLE
Added ThreeJS category

### PR DESCRIPTION
**Pull request by ArnabDey1543**
1. threejs official documentation
2.  threeJS tutorials by Bruno simon
3. free youtube courses by DesignCourse

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1104,6 +1105,18 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 **[⬆ Back to Index](#content)**
 
+## ThreeJS
+
+- :file_folder: [ThreeJS Documentation](https://threejs.org/docs/index.html#manual/introduction/Creating-a-scene)
+- :file_folder: [ThreeJS Resources](https://threejs.org/manual/#en/fundamentals)
+- :video_camera: [Threejs Tutorial Crash Course](https://www.youtube.com/watch?v=YK1Sw_hnm58)
+- :video_camera: [ThreeJS Course: threeJs journey by Bruno Simon](https://threejs-journey.com/)
```
Comment (shahednasser) [inline README.md:1113]:
```suggestion
- :books: [Three.js Course: Three.js journey by Bruno Simon](https://threejs-journey.com/)
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your contribution. In addition to the comment I left below, please make sure you use the name "Three.js" in all headings, links, etc...

File: README.md (hunk 1 above)
Comment (ArnabDey1543) [reply README.md:1113]: Sure, i'll make the changes

**Comment by ArnabDey1543** [comment]
Ma'am i've made the changes as you asked, will also try add some more top notch resources to Three.js category.
I am also thinking of adding AWS cloud category with services in subcategories.

**Comment by ArnabDey1543** [comment]
@shahednasser ma'am i have resolved all the issues, please consider my PR

File: README.md [hunk 2]
```diff
@@ -1104,6 +1105,18 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 **[⬆ Back to Index](#content)**
 
+## Three.js
+
+- :file_folder: [Three.js Documentation](https://threejs.org/docs/index.html#manual/introduction/Creating-a-scene)
+- :file_folder: [Three.js Resources](https://threejs.org/manual/#en/fundamentals)
+- :video_camera: [Three.js Tutorial Crash Course](https://www.youtube.com/watch?v=YK1Sw_hnm58)
+- :video_camera: [Three.js Course: Three.js journey by Bruno Simon](https://threejs-journey.com/)
```
Comment (shahednasser) [inline README.md:1113]:
```suggestion
- :books: [Three.js Course: Three.js journey by Bruno Simon](https://threejs-journey.com/)
```

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Pull request closed without merging**